tests/unit/conftest.py

```diff
@@ -0,0 +1,25 @@
+from datetime import datetime
+from random import randint
+from string import ascii_letters
+
+import pytest
+
+IMG_URL = "https://picsum.photos/200/300"
+
+
+@pytest.fixture
+def rand_gen():
+
+    def gen(field_type):
+        if field_type is str:
+            return "".join(ascii_letters[randint(0,
+                                                 len(ascii_letters) - 1)]
+                           for _ in range(16))
+
+        if field_type is datetime:
+            return datetime.now()
+
+        raise Exception("Can't random generate for field type '%r'" %
+                        field_type)
+
+    return gen
```

Comment:
Since this is used by both unit and integration tests, you can just put a conftest.py under tests/ that contain the shared functions.